activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -44,7 +44,7 @@ def <<(*word)
         end
 
         def add(words)
-          words = words.flatten.map(&:downcase)
+          words = words.flatten.map(&:downcase).reject { |word| uncountable?(word) }
```

Comment:
I checked if changing the parent class is OK, but it seems to be unsafe to do so. `uncountables` is currently a public method and is expected to return Array like value.
(tested here: https://github.com/rails/rails/blob/master/activesupport/test/inflector_test.rb#L503)

So it is better to keep the parent class as is.